src/main/java/model/LottoNumber.java

```diff
@@ -0,0 +1,67 @@
+package model;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.IntStream;
+
+public class LottoNumber {
+    private static final int ZERO = 0;
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final String LOTTO_NUMBER_MUST_BE_POSITIVE = "로또 번호는 양수여야 합니다.";
+    private static final String LOTTO_NUMBER_MUST_BETWEEN_MIN_LOTTO_NUMBER_AND_MAX_LOTTO_NUMBER = "로또 번호는 1과 45 사이의 숫자여야 합니다.";
+
+    private static final Map<Integer, LottoNumber> CACHE = new HashMap<>();
+
+    static {
+        IntStream.rangeClosed(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)
+                .forEach(number -> CACHE.put(number, new LottoNumber(number)));
+    }
+
+    private final int number;
+
+    public LottoNumber(int number) {
```

Comment:
```suggestion
    private LottoNumber(int number) {
```

LottoNumber를 캐싱했기 때문에 외부에서는 생성자로 LottoNumber를 새로 생성하는 것이 아닌 ``getLottoNumberFromCache``로 캐싱 값을 가져오도록 변경하는 것이 좋을 것 같아요. 생성자를 public으로 열어두는 것 보다는 private로 외부에서 사용하지 못하게 닫아두는 것이 의도한대로 LottoNumber를 사용하도록 강제할 수 있어요.

Reply:
생각지도 못한 부분인 것 같습니다. 피드백 감사합니다.